core/dnsserver/config.go

```diff
@@ -56,16 +56,16 @@ type Config struct {
 //after server is started ONLY, can be used as a Key for identifing that config
 // :53 or 127.0.0.1:53 or 127.0.0.1:53/::1:53
 func (c *Config) HostAddresses() string {
-	hosts := ""
+	all := ""
 	for _, h := range c.ListenHosts {
-		host := net.JoinHostPort(h, c.Port)
-		if hosts == "" {
-			hosts = host
+		addr := net.JoinHostPort(h, c.Port)
+		if all == "" {
```

Comment:
only changed the names of the variables,.
what I test as empty s "all" in order to correctly insert the separator of addresses.